app/src/main/java/com/spendster/presentation/signUp/SignUpActivity.java

```diff
@@ -10,34 +10,32 @@
 
 import com.spendster.BuildConfig;
 import com.spendster.R;
+import com.spendster.presentation.AuthView;
 import com.spendster.presentation.moreAboutYou.MoreAboutYouActivity;
 
-public class SignUpActivity extends AppCompatActivity implements SignUpView {
+public class SignUpActivity extends AppCompatActivity implements AuthView {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_sign_up);
 
-        final SignUpPresenter signUpPresenter = new SignUpPresenter(this, new ServerSignUpModel());
         final Button btnContinue = findViewById(R.id.btnContinue);
         final EditText etEmail = findViewById(R.id.etEmail);
         final EditText etPassword = findViewById(R.id.etPassword);
         final EditText etRetypePassword = findViewById(R.id.etRetypePassword);
+        final EditText etUserName = findViewById(R.id.etUserName);
 
-        if(BuildConfig.DEBUG){
-            etEmail.setText("test@gmail.com");
-            etPassword.setText("12345678");
-            etRetypePassword.setText("12345678");
-        }
+        final SignUpPresenter signUpPresenter = new SignUpPresenter(this, new ServerSignUpModel());
 
         btnContinue.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
                 String email = etEmail.getText().toString();
                 String password = etPassword.getText().toString();
                 String retypePassword = etRetypePassword.getText().toString();
-                signUpPresenter.signUp(email, password, retypePassword);
+                String username = etUserName.getText().toString();
```

Comment:
@SimonMaha already mentioned about extracting such lines into method and making a trim of text